app/components/inspector/SimulatorCard.tsx

```diff
@@ -49,15 +53,22 @@ export function SimulatorCard({ message }: { message: VersionedMessage }) {
     }
 
     return (
-        <div className="card">
-            <div className="card-header">
-                <h3 className="card-header-title">Transaction Simulation</h3>
-                <button className="btn btn-sm d-flex btn-white" onClick={simulate}>
-                    Retry
-                </button>
+        <>
+            <div className="card">
+                <div className="card-header">
+                    <h3 className="card-header-title">Transaction Simulation</h3>
+                    <button className="btn btn-sm d-flex btn-white" onClick={simulate}>
+                        Retry
+                    </button>
+                </div>
+                <ProgramLogsCardBody message={message} logs={logs} cluster={cluster} url={url} />
             </div>
-            <ProgramLogsCardBody message={message} logs={logs} cluster={cluster} url={url} />
-        </div>
+            {showTokenBalanceChanges && simulationTokenBalanceRows && !simulationError && simulationTokenBalanceRows.rows.length && (
+                <div className="card">
+                    <TokenBalancesCardInner rows={simulationTokenBalanceRows.rows} />
+                </div>
+            )}
```

Comment:
```suggestion
            {showTokenBalanceChanges && simulationTokenBalanceRows && !simulationError && simulationTokenBalanceRows.rows.length ? (
                <div className="card">
                    <TokenBalancesCardInner rows={simulationTokenBalanceRows.rows} />
                </div>
            ) : null}
```

I think this needs to be using a ternary. On a transaction with no simulation rows (but also no error), for example `Afs79yHxq/0meJcx8g8Ii5TgxqqzKiRfpn7PFKZjKK82h0zUovJyqjlCQ4LZlY1qyJxUiHfrk+PWlmR+0n1cRAyAAQADB+Stj1FYmqtHlZI/QGmQeYoR1irFeTLBI2sp7Gc6ug7MuCuUoNYwatIHcVDwZphNsSdPcn3cWj7QDruFzkL8aV0PqJUEtyjqPaTVYdCQwoy3TlS7gGgNtvZHVS5ljgMmfg7CBa/RYA94N/8Dm0+91HOmbMuq3k5VX9oBUUprCZcLAwZGb+UhFzL/7K26csOb57yM5bvF9xJrLEObOkAAAAAIQcuVuAJV1WUsDbUNQYARXuUf1y8xSDmekMHgzfF4Tozl/UQw28pzeUpQzoB+d1FwIc8XGJJMTrf86rvH9XywCMULIemm/r4y+W1vlqddqaEK5+j2ZJjP3MHekr417mUMBAAJA+gDAAAAAAAABQoAEgETFAIHCAkKCYu+5vlNoM4EAQUZEhUBABMWFAIXBwsKCAkMDQ4PDBgZGhAGAy1b9mAHNRbq4WipWwUGAAAACwAAAAAAAAABBAFu9OSOigEAAAEDAAAAU0RLAAEFGRIVAQATFhQCFwcLCggJDw0ODwwYGRoRBgMtW/ZgBzUW6uFwj000BgAAAAsAAAAAAAAAAgQBb/TkjooBAAABAwAAAFNESwABBRkSFQEAExYUAhcHCwoICQwNDg8MGBkaEAYDLVv2YAc1FurhGLum/QUAAAAPAAAAAAAAAAEEAXD05I6KAQAAAQMAAABTREsAAQUZEhUBABMWFAIXBwsKCAkPDQ4PDBgZGhEGAy1b9mAHNRbq4XiYPjwGAAAADwAAAAAAAAACBAFx9OSOigEAAAEDAAAAU0RLAAEFGRIVAQATFhQCFwcLCggJDA0ODwwYGRoQBgMtW/ZgBzUW6uEIQMz5BQAAABMAAAAAAAAAAQQBcvTkjooBAAABAwAAAFNESwABBRkSFQEAExYUAhcHCwoICQ8NDg8MGBkaEQYDLVv2YAc1FurhCBs3QAYAAAATAAAAAAAAAAIEAXP05I6KAQAAAQMAAABTREsAAQUZEhUBABMWFAIXBwsKCAkMDQ4PDBgZGhAGAy1b9mAHNRbq4eDI8fUFAAAAFgAAAAAAAAABBAF09OSOigEAAAEDAAAAU0RLAAEFGRIVAQATFhQCFwcLCggJDw0ODwwYGRoRBgMtW/ZgBzUW6uGYnS9EBgAAABYAAAAAAAAAAgQBdfTkjooBAAABAwAAAFNESwABBRkSFQEAExYUAhcHCwoICQwNDg8MGBkaEAYDLVv2YAc1FurhuFEX8gUAAAAaAAAAAAAAAAEEAXb05I6KAQAAAQMAAABTREsAAQUZEhUBABMWFAIXBwsKCAkPDQ4PDBgZGhEGAy1b9mAHNRbq4SggKEgGAAAAGgAAAAAAAAACBAF39OSOigEAAAEDAAAAU0RLAAEBYQWxzthEkoKua2gEhVWJDudkqzKq8gXk22w8tDHQoVQLHB8gHh0kISIjJSYJAAIEAQMFCQoG` on mainnet, I'm getting a 0 rendered:

<img width="1155" alt="Screenshot 2023-09-13 at 16 55 07" src="https://github.com/solana-labs/explorer/assets/1711350/65fb989e-8dc6-453a-a9db-3493743e6dda">